gc/base/standard/ConcurrentGCSATB.cpp

```diff
@@ -107,12 +107,10 @@ MM_ConcurrentGCSATB::kill(MM_EnvironmentBase *env)
 void
 MM_ConcurrentGCSATB::tearDown(MM_EnvironmentBase *env)
 {
-#if defined(OMR_GC_REALTIME)
 	if (NULL != _extensions->sATBBarrierRememberedSet) {
 		_extensions->sATBBarrierRememberedSet->kill(env);
 		_extensions->sATBBarrierRememberedSet = NULL;
 	}
-#endif /* defined(OMR_GC_REALTIME) */
```

Comment:
Removing `if defined(OMR_GC_REALTIME)` here required all variables inside to be defined out of `OMR_GC_REALTIME` too.
So definition in `GCExtensionsBase.hpp` for `sATBBarrierRememberedSet` should be defined out of `OMR_GC_REALTIME` but I can not see this change. Did I miss it?
If it is a problem would you please check how other variables are defined?